SCons/Environment.xml

```diff
@@ -1901,123 +1901,148 @@ Nodes or strings representing path names.
-the resulting expanded pattern.
+and returns whatever matches the resulting expanded pattern.
+The results are sorted, unlike for the similar &Python;
+<systemitem>glob.glob</systemitem> function,
+to ensure build order will be stable.
 </para>
 
 <para>
-The specified
 <parameter>pattern</parameter>
-uses Unix shell style metacharacters for matching:
+can contain POSIX-style shell metacharacters for matching:
+</para>
+
+<informaltable rowsep="1" colsep="1" frame="topbot">
+<tgroup cols="2">
+<thead>
+  <row>
+    <entry>Pattern</entry>
+    <entry>Meaning</entry>
+  </row>
+</thead>
+<tbody>
+  <row>
+    <entry><literal>*</literal></entry>
+    <entry>matches everything</entry>
+  </row>
+  <row>
+      <entry><literal>?</literal></entry>
+      <entry>matches any single character</entry>
+  </row>
+  <row>
+      <entry><literal>[seq]</literal></entry>
+      <entry>matches any character in <emphasis>seq</emphasis>
+      (can be a list or a ranage).</entry>
+  </row>
+  <row>
+      <entry><literal>[!seq]</literal></entry>
+      <entry>matches any character not in <emphasis>seq</emphasis></entry>
+  </row>
+</tbody>
+</tgroup>
+</informaltable>
+
+<para>
+For a literal match, wrap the metacharacter in brackets to
+escape the normal behavior.
+For example, <literal>'[?]'</literal> matches the character
+<literal>'?'</literal>.
+</para>
+
+<para>
+Filenames starting with a dot are specially handled -
+they can only be matched by patterns that start with a dot
+(or have a dot immediately following a pathname separator
+character, or slash), they are not not matched by the metacharacters.
+Metacharacter matches also do not span directory separators.
 </para>
 
-<example_commands>
-  *       matches everything
-  ?       matches any single character
-  [seq]   matches any character in seq
-  [!seq]  matches any char not in seq
-</example_commands>
-
 <para>
-If the first character of a filename is a dot,
-it must be matched explicitly.
-Character matches do
-<emphasis>not</emphasis>
-span directory separators.
-</para>
-
-<para>
-The
 &f-Glob;
-knows about
-repositories
+understands repositories
 (see the
 &f-link-Repository;
 function)
 and source directories
 (see the
 &f-link-VariantDir;
 function)
-and
-returns a Node (or string, if so configured)
+and returns a Node (or string, if so configured) match
 in the local (SConscript) directory
 if a matching Node is found
 anywhere in a corresponding
 repository or source directory.
 </para>
 
 <para>
-The
+If the optional
 <parameter>ondisk</parameter>
-argument may be set to a value which evaluates
-<constant>False</constant>
-to disable the search for matches on disk,
-thereby only returning matches among
-already-configured File or Dir Nodes.
-The default behavior is to
-return corresponding Nodes
-for any on-disk matches found.
+argument evaluates false,
+the search for matches on disk is disabled,
+and only matches from
+already-configured File or Dir Nodes are returned.
```

Comment:
I'd keep the explanation of the default (True) setting?

Reply:
Restored (in slighlly different form)